### PR TITLE
Spellcheck the repo

### LINKED ISSUE #164: Spell check the repo
> let's file an issue to come back and spell check everything? No reason not to have the other stuff checked as well. Maybe not the cheatsheets, but maybe we are fine with some random partial words being added.

_Originally posted by @jashapiro in https://github.com/AlexsLemonade/reproducible-research/pull/163#pullrequestreview-2036494656_

--- 

As it says above, let's do a comprehensive spell check.

### PR DESCRIPTION
**Pull request by sjspielman**
Closes #164 

Spellchecking the repo! Filing as a draft to first catch and fix the spelling errors.

### CODE REVIEW

**Comment by sjspielman** [comment]
This is now ready for review. As part of review, please weigh in on:

- Do we want to keep the GHA checking all these files? (maybe we can remove the `files` argument altogether at this point?)
- If we do _not_ want to do ^, I could also remove the partial words from the dictionary, e.g. `ual` derived from **man**ual.

File: .github/workflows/spellcheck.yml [hunk 1]
```diff
@@ -28,7 +28,7 @@ jobs:
         id: spell
         with:
           dictionary: components/dictionary.txt
-          files: "docs/* docs/*/*"
+          files: "docs/* docs/*/* workshop_materials/cheatsheets/* instructor_notes/* instructor_notes/openscpca/* "
```
Comment (jashapiro) [inline .github/workflows/spellcheck.yml:31]: I'd probably just remove this at this point and check everything. I'm not too worried about the partial words added being a problem.

I also thought of another thing we could  if we want to exclude particular files: add a step with `rm` for the files we want the spellchecker to skip.

File: components/dictionary.txt [hunk 2]
```diff
@@ -100,13 +124,16 @@ pre
...
 renv
-Repo
+repo
 reproducibility
 rerio
+rint
 Rittenhouse
 Riverol
 Rohan
+Rscript
```
Comment (jashapiro) [inline components/dictionary.txt:134]: I would expect we usually want this styled as `Rscript` so maybe it isn't needed here?

File: instructor_notes/06_shell-scripting.md [hunk 3]
```diff
@@ -17,7 +17,7 @@ At the end of this activity, workshop attendees should be able to:
 
 The goal of this activity is to interactively develop a script to download and save these two paired FASTQ files:
 
-+ Paired FASTQ files from [Study ID SRP255885](https://trace.ncbi.nlm.nih.gov/Traces/sra/?study=SRP255885), and RNA-seq dataset about medulloblastoma
++ Paired FASTQ files from [Study ID `SRP255885`](https://trace.ncbi.nlm.nih.gov/Traces/sra/?study=SRP255885), and RNA-seq dataset about Medulloblastoma
```
Comment (jashapiro) [inline instructor_notes/06_shell-scripting.md:20]:
```suggestion
+ Paired FASTQ files from [Study ID `SRP255885`](https://trace.ncbi.nlm.nih.gov/Traces/sra/?study=SRP255885), an RNA-seq dataset about medulloblastoma
```

File: components/dictionary.txt [hunk 4]
```diff
@@ -4,8 +4,11 @@ ACH
 Addgene
 al
 ALSF
+AnnData
 authenticator
 Authy
+Autocomplete
```
Comment (jashapiro) [inline components/dictionary.txt:10]:
```suggestion
autocomplete
```

File: components/dictionary.txt [hunk 5]
```diff
@@ -25,26 +28,32 @@ chatbot
...
 endcomment
 endif
 endfor
 Ensembl
 epitope
+errored
 et
 familiaris
+fastq
+FASTQ
```
Comment (jashapiro) [inline components/dictionary.txt:56]: I would check how fastq/FASTQ are being used, as I would generally expect us to want to standardize on one.

**Review by jashapiro — APPROVED** [review]
LGTM. I'd probably just check all files to keep things simpler. I suggested another way to exclude files from checks if we want to do that.

Othere than that, just a few minor comments.

File: instructor_notes/fastq_subset/README.md [hunk 6]
```diff
@@ -1,3 +1,3 @@
 These FASTQ files are subsetted versions (to only the first 10,000 reads) of two of the samples from this project: https://trace.ncbi.nlm.nih.gov/Traces/sra/?study=SRP255885.
 
-Files are obtained from EBI and processed with `generate_fastq_subsets.sh`, and they can be used as alternative FASTQ files to download when internet speed is prohibitive.
+Files are obtained from `EBI` and processed with `generate_fastq_subsets.sh`, and they can be used as alternative FASTQ files to download when internet speed is prohibitive.
```
Comment (jashapiro) [inline instructor_notes/fastq_subset/README.md:3]: EBI is an abbreviation, not code. Also tense change...
```suggestion
Files were obtained from EBI and processed with `generate_fastq_subsets.sh`, and they can be used as alternative FASTQ files to download when internet speed is prohibitive.
```

Comment (sjspielman) [reply instructor_notes/fastq_subset/README.md:3]: alright, fine.

**Pull request merged**